src/Common/error-handler-utils.ts

```diff
@@ -0,0 +1,40 @@
+import { Error as grpcWebError } from 'grpc-web'
```

Comment:
My thinking here was that this class would be for all network client error handling.  If you and @amiecorso ever wanted to handle gRPC errors coming from your rippled node, you could do so in here.  I think there are probably similarities between error handling for ILP and XRP/PayID calls that would need to be pulled into a common class anyway, so why not handle everything in a common class?

Reply:
So, this seems ILP specific to me because it returns an ILP error. I’d prefer we left this in ILP for now and refactored it when it is actually generic. 

Otherwise common sort of becomes a dumping ground of things that aren’t actually common.

Reply:
Fair point,  will move it to ILP